### PR TITLE
docs: add getting started section and remove outdated docs

### LINKED ISSUE #267: ModuleNotFoundError: No module named 'unstructured.documents.pdf'
Hello,

Following the documentation for parsing a pdf as found here: https://unstructured-io.github.io/unstructured/examples.html#pdf-parsing

It seems that the import statement:
`from unstructured.documents.pdf import PDFDocument
`
results in a not found error.

Indeed, checking unstructured/unstructured/documents, I can't seem to find anything relevant for PDF parsing.

Thank you

### PR DESCRIPTION
**Pull request by MthwRobinson**
### Summary

Closes #267. Removes outdated documentation from the examples section. Also adds a getting started section to highlight basic workflows. Updates the training notebook to use `convert_to_dict` instead of `convert_to_isd`.

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+* Partitiong a document with the ``partition`` function.
+* Understand how documents are structured in ``unstructured``.
+* Convert a document to a dictionary and/or save it as a JSON.
+
+The example documents in this section come from the
+`example-docs <https://github.com/Unstructured-IO/unstructured/tree/main/example-docs>`_
+directory in the ``unstructured`` repo.
+
+Before running the code in this make sure you've installed the ``unstructured`` library
+and all dependencies using the instructions in the quick start section.
```
Comment (cragwolfe) [inline docs/source/getting_started.rst:16]: Quick Start (maybe bold)?

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+
+#######################
+Partitioning a document
+#######################
+
+In this section, we'll cut right to the chase and get to the most important part of the library: partitioning a document.
+The goal of document partitioning is to read in a source document, split the document into sections, categorize those sections,
+and extract the text associated with those sections. Depending on the document type, unstructured uses different methods for
+partitioning a document. We'll cover those in a later training notebook. For now, we'll use the simplest API in the library,
```
Comment (cragwolfe) [inline docs/source/getting_started.rst:26]: "We'll cover those in a later training notebook" -> in a later section?

File: docs/source/getting_started.rst [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+.. code:: python
+
+	with open("example-10k.html", "rb") as f:
+	    elements = partition(file=f)
+
+
+The ``partition`` function uses `libmagic <https://formulae.brew.sh/formula/libmagic>`_ for filetype detection. If ``libmagic`` is
+not present and the user passes a filename, ``partition`` falls back to detecting the filetype using the file extension.
+``libmagic`` is required if you'd lke to pass a file-like object to ``partition``.
```
Comment (cragwolfe) [inline docs/source/getting_started.rst:50]: Add a disclaimer that very different behaviors may be observed with and without libmagic?

File: docs/source/getting_started.rst [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+* ``Element``
+	* ``Text``
+		* ``FigureCaption``
+		* ``NarrativeText``
+		* ``ListItem``
+		* ``Title``
+		* ``Address``
+		* ``PageBreak``
+	* ``CheckBox``
+	* ``Image``
```
Comment (cragwolfe) [inline docs/source/getting_started.rst:72]: PageBreak?

File: docs/source/getting_started.rst [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+
+Other element types that we will add in the future include tables and figures.
```
Comment (cragwolfe) [inline docs/source/getting_started.rst:75]: switch to passive voice, "that will be added in the future"?

**Review by cragwolfe — APPROVED** [review]
LGTM with some minor nits

File: examples/training/0-Core Concepts.ipynb [hunk 6]
```diff
@@ -274,7 +274,7 @@
    "source": [
     "## Converting to a dictionary <a id=\"dict\"></a>\n",
     "\n",
-    "The final step in the process for most users is to convert the output to JSON. You can convert a list of document elements to a list of dictionaries using the `convert_to_isd` function. ISD stands for \"initial structured data\", our common format for representing text data. The workflow for using `convert_to_isd` appears below."
+    "The final step in the process for most users is to convert the output to JSON. You can convert a list of document elements to a list of dictionaries using the `convert_to_isd` function. The workflow for using `convert_to_isd` appears below."
```
Comment (qued) [inline examples/training/0-Core Concepts.ipynb:277]: Shouldn't these refs to `convert_to_isd` become `convert_to_dict`?

File: docs/source/getting_started.rst [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+Different partioning functions use different methods for determining the element type and extracting the associated content.
```
Comment (qued) [inline docs/source/getting_started.rst:76]:
```suggestion
Different partitioning functions use different methods for determining the element type and extracting the associated content.
```

File: docs/source/getting_started.rst [hunk 8]
```diff
@@ -0,0 +1,159 @@
+Getting Started
+===============
+
+The following section will cover basic concepts and usage patterns in ``unstructured``.
+After reading this section, you should be able to:
+
+* Partitiong a document with the ``partition`` function.
```
Comment (qued) [inline docs/source/getting_started.rst:7]:
```suggestion
* Partition a document with the ``partition`` function.
```

**Review by qued — APPROVED** [review]
LGTM (pending nits)!

**Comment by cragwolfe** [comment]
Related: probably makes sense to create an issue so that CI actually tests doc code snippets for correctness.

**Pull request merged**